internal/buildscripts/packaging/msi/splunk-otel-collector.wxs

```diff
@@ -1,6 +1,6 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
    <Product Id="*" UpgradeCode="fde3e4d9-9ca5-4c82-be7b-81445ab5b605" Name="Splunk OpenTelemetry Collector" Version="$(var.Version)" Manufacturer="Splunk, Inc." Language="1033">
-      <Package InstallerVersion="200" Compressed="yes" Comments="Windows Installer Package"/>
+      <Package InstallerVersion="200" Compressed="yes" Comments="Windows Installer Package" InstallPrivileges="elevated" InstallScope="perMachine" />
```

Comment:
Does this mean that it can only be installed/started by a machine admin? I am wondering if we should have the default set to the equivalent of `ALLUSERS=2` instead. From the docs, `The value ALLUSERS=2 enables the system to reset the value of ALLUSERS, and the installation context, dependent upon the user's privileges and the version of Windows.`. I gather this would mean that if an admin installs it, the scope will be per-machine, but a normal user install per-user which will make this work for both cases?

Reply:
Good Q. Your reading is the same as mine, however, a non-admin user can't install a Windows service, so in practice the MSI already requires admin to install the collector, so there is no practical value in our case.